alcs-frontend/src/app/features/search/search.component.html

```diff
@@ -79,6 +79,65 @@ <h4>File Details</h4>
         </mat-form-field>
       </div>
     </div>
+    <div class="row">
+      <div class="column">
+        <div
+          class="tag-container"
+          [ngClass]="{ hovered: hovered, clicked: clicked }"
+          (mouseleave)="hovered = false"
+          (mouseenter)="hovered = true"
+          (click)="onClick()"
+        >
+          <mat-form-field class="tag-field" appearance="outline">
+            <mat-label>Tags</mat-label>
+            <mat-chip-grid #chipGrid>
+              <app-tag-chip *ngFor="let tag of tags" (removeClicked)="removeTag(tag)" [tag]="tag"></app-tag-chip>
+              <input
+                #tagInput
+                [matChipInputFor]="chipGrid"
+                [formControl]="tagControl"
+                [matAutocomplete]="auto"
+                [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
+                (focus)="autoCompleteTrigger.openPanel()"
+                (click)="clearSearch()"
+                (blur)="checkDirty()"
+              />
+              @if (tags.length > 0) {
```

Comment:
You're using *ngFor in the bottom and @if here which is the new template syntax. Was there any specific reason for it?